python_ta/util/autoformat.py

```diff
@@ -0,0 +1,40 @@
+"""
+Utility class for running the Black formatting tool.
+"""
+
+import subprocess
+import sys
+from typing import Optional
+
+
+class Autoformatter:
```

Comment:
Overall the refactoring is on the right track, but it's a bit overkill to create a separate class. You can just define a function (`run_autoformat`) that contains the logic for taking in the configuration options and running Black.